easybuild/easyconfigs/t/TAU/TAU-2.22.2-gompi-1.4.12-no-OFED.eb

```diff
@@ -0,0 +1,34 @@
+# This is an easyconfig file for EasyBuild, see https://github.com/hpcugent/easybuild
+# Copyright:: Copyright 2013 Juelich Supercomputing Centre, Germany
+# Authors::   Bernd Mohr <b.mohr@fz-juelich.de>
+# License::   New BSD
+#
+# This work is based from experiences from the UNITE project
+# http://apps.fz-juelich.de/unite/
+##
+name = 'TAU'
+version = '2.22.2'
+
+homepage = 'http://tau.uoregon.edu'
+description = """The TAU Performance System is a portable profiling and tracing toolkit
+ for performance analysis of parallel programs written in Fortran, C, C++, Java, Python."""
+
+toolchain = {'name': 'gompi', 'version': '1.4.12-no-OFED'}
+toolchainopts = {'usempi': True}
+
+source_urls = ['http://www.cs.uoregon.edu/research/paracomp/tau/tauprofile/dist/']
+sources = [SOURCELOWER_TAR_GZ]
+
+dependencies = [
+    ('OTF', '1.12.4'),
+    ('PAPI', '5.2.0'),
+    ('PDT', '3.19'),
+    ('Scalasca', '1.4.3'),
+    ('Score-P', '1.2.1'),
```

Comment:
Thanks for the feedback @berndmohr, I'll go ahead and comment out `Scalasca` & `VampirTrace` here as dependencies, mentioning the comment you suggest, so I can get this (and especially the easyblock for Tau @ https://github.com/hpcugent/easybuild-easyblocks/pull/887) merged in.